TechAndCheck/zenodotus
# Allow users to reset their passwords

This PR enables the Devise password reset workflow in Zenodotus. 

## Testing instructions
1. Create a Zenodotus account with a live email address you control (or change an existing Zenodotus account to use such an address)
2. On the login page, click the reset password link. Enter your email address on the reset page. 
3. Find the Zenodotus password reset email in your inbox (or spam folder). Click the rest link in the email (you may need to add the Zenodotus server port number to the URL)
4. Try resetting your password with:
  a. A 6+ character password that you do not confirm
  b. A <6 character password that you properly confirm
  c. A password of any length that you confirm incorrectly
5. Each of the password reset attempts above should fail and lead to corrective instructions being displayed. To actually reset your password, enter a phrase  of 6+ characters and confirm it properly. 
6. Try to log in again with your newly changed  password  

Closes #297
